wdio-android-app/package.json

```diff
@@ -15,13 +15,14 @@
     "typescript": "^5.4.3"
   },
   "scripts": {
-    "sauce-visual": "wdio run ./tests/configs/wdio.saucelabs.mobile.conf.ts --spec tests/specs/visual.spec.ts",
-    "sauce-visual-check": "VISUAL_CHECK=true wdio run ./tests/configs/wdio.saucelabs.mobile.conf.ts --spec tests/specs/visual.spec.ts",
+    "sauce-visual": "FPS=true wdio run ./tests/configs/wdio.saucelabs.mobile.conf.ts --spec tests/specs/visual.spec.ts",
```

Comment:
do we want to have it under an env var? This is an example repo. I would say its fine to have it without it

Reply:
yes, because vdc is running same test file and it's not working in vdc yet